app/taskqueue/celeryconfig.py

```diff
@@ -18,7 +18,7 @@
 
 """Celery configuration values."""
 
-BROKER_URL = "redis://localhost/0"
+BROKER_URL = "redis://redis/0"
```

Comment:
Err, this doesn't seem related to the kcidb change.

Reply:
oops, that's unrelated.  Removed.